indico/modules/events/papers/templates/display/call_for_papers.html

```diff
@@ -0,0 +1,99 @@
+{% extends 'layout/conference_page_base.html' %}
+
+{% from 'events/management/_lists.html' import render_displayed_entries_fragment %}
+{% from 'events/papers/display/_common.html' import render_cfp_infoline %}
+{% from 'events/papers/display/_paper_list.html' import render_user_papers_list %}
+
+{% block title %}
+    {%- trans %}Call for Papers{% endtrans -%}
+{% endblock %}
+
+{% macro render_timestamp(label, date, tz, completed=false, alternative_text=none) %}
```

Comment:
> And what should I call the argument representing CFP or CFA that can be passed there ?  

I think we are calling them _Call for proposals_ as stated here: https://github.com/indico/indico/blob/master/indico/modules/events/models/reviews.py#L39
Not 100% sure though.